tests/js/Dockerfile

```diff
@@ -14,8 +14,8 @@ COPY package*.json ./
 RUN npm ci
 
 # Copy over only the folders needed for our tests.
-COPY ../../playwright.config.js ./
+COPY playwright.config.js ./
```

Comment:
I'm not sure why I had the `../../` before. It was working, but isn't consistent with how I did things elsewhere.